src/index.js

```diff
@@ -20,13 +20,24 @@ const middlewareList = process.env.NODE_ENV === 'development' ?
   [sagaMiddleware, logger] :
   [sagaMiddleware];
 
-const store = createStore(
-  // tells the saga middleware to use the rootReducer
-  // rootSaga contains all of our other reducers
-  rootReducer,
-  // adds all middleware to our project including saga and logger
-  applyMiddleware(...middlewareList),
-);
+const enhancers = []
+// this allows you to use
+// Redux dev tools from the chrome store.
+if (process.env.NODE_ENV === 'development') {
+    // tslint:disable-next-line
+    const devToolsExtension = (window).devToolsExtension
```

Comment:
https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en

This allows me to browse what's happening in redux so I can create selectors and connect it to stateless components.

1. Pull this PR
2. Install Chrome devtool
3. start and load the website  into chrome
4. There is now a "Redux" tab to the right of Elements, Console, Network, etc.

![image](https://user-images.githubusercontent.com/1424223/53345058-b22f1700-38d9-11e9-8fa8-0f91c863f570.png)


This makes life easier.